modules/op-release-notes-1-6.adoc

```diff
@@ -0,0 +1,204 @@
+// Module included in the following assembly:
+//
+// * cicd/pipelines/op-release-notes.adoc
+
+[id="op-release-notes-1-6_{context}"]
+= Release notes for {pipelines-title} General Availability 1.6
+
+With this update, {pipelines-title} General Availability (GA) 1.6 is available on {product-title} 4.9.
```

Comment:
The module level table is missing, did we decide against it?

Reply:
I applied your suggestion to remove it.